modules/__tests__/Media-test.js

```diff
@@ -133,7 +133,7 @@ describe('A <Media>', () => {
         )
 
         render(element, node, () => {
-          expect(node.firstChild.innerHTML).toNotMatch(/hello/)
+          expect(node.firstChild.innerHTML).not.toBeDefined()
```

Comment:
Interesting... Not sure if it's a bug or not.

You can do `toBeUndefined()` instead of `.not.toBeDefined()`, btw.

Green build though, woo!